src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -209,14 +209,17 @@ const CreateOrEditQuizContainer = ({
         isCreationOpen ? t('myResourcesPage.quizzes.savePreviousQuestion') : ''
       }
     >
-      <AppButton
-        onClick={onOpenCreateQuestion}
-        size={SizeEnum.ExtraLarge}
-        variant={ButtonVariantEnum.Tonal}
-      >
-        {t('myResourcesPage.quizzes.createNewQuestion')}
-        <EditIcon fontSize={SizeEnum.Small} />
-      </AppButton>
+      <span>
```

Comment:
Maybe we should finally refactor AppButton to accept a ref? 
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/109922101/472f75ee-96bd-4d41-b644-d6645aaf29c1)

As you can see from mui docs, simple Button works here. We can avoid dom elements nesting with such approach

Reply:
I did such an experiment) After adding ref and forwardRef to AppButton without <span> error disappeared but I can't add disabled={isCreationOpen} and it is not wery well user experience.... And also variant={ButtonVariantEnum.Tonal} doesn't work((((


https://github.com/ita-social-projects/SpaceToStudy-Client/assets/127191734/fbb20d30-cbaa-4f29-b1f3-b4c30153d6ba

Reply:
I think there might be another problem related to it because ref should not impact on styles

Reply:
I've understood why styles is broken after adding tooltip to the AppButton !
The main reason isn't forwardRef, of course)
This button, after wrapping in Tooltip, has property data-mui-internal-clone-element='true': This attribute indicates that the element is being cloned internally by Material-UI. Material-UI sometimes clones elements for various reasons such as event handling, state management, or accessibility enhancements. It's used internally by the library and should generally not be manipulated directly. (our s2s_button styles obviously don't work with cloned elements)

![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/127191734/3f3b5e53-ffc0-4ed4-b285-735fe1d886c3)

And when we add <span> it prevents manipulating of the nested button elemetn by Tooltip
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/127191734/1cea305a-c43a-47df-85f1-d535276a7495)

In that case <span> works as a element that allows as add Tooltip to castom element without manipulating of this element